mypy/plugins/enums.py

```diff
@@ -17,10 +17,12 @@
 from typing import TypeVar, cast
 
 import mypy.plugin  # To avoid circular imports.
-from mypy.nodes import TypeInfo
+from mypy.checker import TypeChecker
```

Comment:
It's a bit unfortunate that we have another circular import dependency. What about adding adding a helper method for checking whether we are type checking a stub file to the checker api? (It's okay to do this in a follow-up PR.)